### ISSUE TITLE
Whenever I refresh the page I get "document is not defined"

### ISSUE THREAD

**Issue by iWanheda** [sindresorhus/screenfull#199]
![image](https://user-images.githubusercontent.com/34320875/147859265-1a18ec3e-fd77-4aac-b1c6-1e04718b4eb2.png)

My code is literally:
```
useEffect(() => {
    if (vidState.fullscreen) {
      if (screenfull.isEnabled) {
        screenfull.request();
      }
    }
  }, [vidState.fullscreen]);```

**Comment by iWanheda** [comment]
Even if I just import it I get the error, without even using it!

**Comment by RyanWarner** [comment]
@iWanheda I solved this by downgrading to version `5.1.0`

See https://github.com/sindresorhus/screenfull/issues/194

**Comment by MajorBreakfast** [comment]
The problem also exists when doing SSR in SvelteKit. While screenfull can't offer functionality on the server, it could be inert, and not throw on import.
